doc/source/change_proposal_process.rst

```diff
@@ -80,6 +70,9 @@ These steps are necessary to initialize a project repo enabling you to contribut
 10. You can check any time with “git status” for pending changes. They
     are marked red if you either modified existing files (“modified”)
     or introduced new ones (“untracked files”).
+    
```

Comment:
I would also add here, that step 9 can be repeated for any file that need to be changed. Additionally this is not really about rst. Here the doc start being too much rst specific, while we want to describe how to push ANY change

Reply:
yes good point a broader approach would be better suiting. 
I will try to be change the next steps to be less specific to rst